ecs-cli/modules/cli/local/down_app.go

```diff
@@ -42,8 +42,30 @@ func Down(c *cli.Context) error {
 		network.Teardown(client)
 	}()
 
+	if err := validateOptions(c); err != nil {
+		logrus.Fatalf("Tasks can be either created by local files or remote files")
```

Comment:
Should this also be `logrus.Fatal(err.Error())`?

Reply:
Yeah I forgot to update here